gateway/app/main/codes.py

```diff
@@ -1,18 +1,44 @@
-class GRID_MSG(object):
-    TYPE_FIELD = "type"
-    DATA_FIELD = "data"
+class MSG_FIELD:
+    TYPE = "type"
+    DATA = "data"
+    WORKER_ID = "worker_id"
+    MODEL = "model"
+    ALIVE = "alive"
+
+
+class CONTROL_EVENTS(object):
     SOCKET_PING = "socket-ping"
-    GET_PROTOCOL = "get-protocol"
+
+
+class WEBRTC_EVENTS(object):
     PEER_LEFT = "webrtc: peer-left"
     INTERNAL_MSG = "webrtc: internal-message"
     JOIN_ROOM = "webrtc: join-room"
 
 
+class FL_EVENTS(object):
+    REPORT = "federated/report"
+    AUTHENTICATE = "federated/authenticate"
+    CYCLE_REQUEST = "federated/cycle-request"
+
+
+class CYCLE(object):
+    STATUS = "status"
+    KEY = "request_key"
+    PING = "ping"
+    DOWNLOAD = "download"
+    UPLOAD = "upload"
+    VERSION = "version"
+    PLANS = "plans"
```

Comment:
Do you need "plans" and "training plans"?

Reply:
Fixed!